MODULE.bazel

```diff
@@ -0,0 +1,71 @@
+module(name = "rules_ros", repo_name = "com_github_mvukov_rules_ros")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "bzip2", version = "1.0.8.bcr.1")
+bazel_dep(name = "eigen", version = "3.4.0.bcr.1")
+bazel_dep(name = "gflags", version = "2.2.2")
+bazel_dep(name = "glog", version = "0.7.0")
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1")
+bazel_dep(name = "lz4", version = "1.9.4")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_foreign_cc", version = "0.10.1")
+bazel_dep(name = "rules_python", version = "0.33.1")
+
+non_module_ros_repositories = use_extension("//extensions:ros_repositories.bzl", "non_module_dependencies")
+use_repo(
+    non_module_ros_repositories,
+    "console_bridge",
+    "roscpp_core",
+    "rosconsole",
+    "ros_genmsg",
+    "ros_gencpp",
+    "ros_genpy",
+    "ros_std_msgs",
+    "ros_comm_msgs",
+    "ros_comm",
+    "ros_ros",
+    "ros_common_msgs",
+    "ros_actionlib",
+    "ros_dynamic_reconfigure",
+    "ros_geometry2",
+    "orocos_kdl",
+    "urdfdom_headers",
+    "tinyxml",
+    "urdfdom",
+)
+
+bazel_dep(name = "rules_boost")
+archive_override(
+    module_name = "rules_boost",
+    urls = "https://github.com/nelhage/rules_boost/archive/42d8155d8f20a1aee8ee20b7903a495bdfb9befd.zip",
+    strip_prefix = "rules_boost-42d8155d8f20a1aee8ee20b7903a495bdfb9befd",
+    integrity = "sha256-I1iTdF3qckTmDRab+0yjA37Iya9AOGvEGJVoPtfpADM=",
+)
+
+non_module_boost_repositories = use_extension("@rules_boost//:boost/repositories.bzl", "non_module_dependencies")
+use_repo(
+    non_module_boost_repositories,
+    "boost",
+)
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
```

Comment:
In my other project rules_ros2, I suggest rules_python should be defined as dev dep here and should be set up in the user repo, https://github.com/mvukov/rules_ros2/pull/238#issuecomment-2171677335.

I'd say the same applies here. We should not constrain our users which Python they are going to use.

Reply:
done